src/components/subscriptions/expiration/SubscriptionExpirationBanner.jsx

```diff
@@ -14,12 +14,12 @@ import ContactCustomerSupportButton from '../buttons/ContactCustomerSupportButto
 
 const SubscriptionExpirationBanner = ({ isSubscriptionPlanDetails }) => {
   const {
-    subscription: { daysUntilExpiration, expirationDate, showExpirationNotifications },
+    subscription: { agreementNetDaysUntilExpiration, expirationDate, showExpirationNotifications },
   } = useContext(SubscriptionDetailContext);
   const [showBanner, setShowBanner] = useState(true);
 
   const renderMessage = () => {
-    const subscriptionExpired = daysUntilExpiration <= 0;
+    const subscriptionExpired = agreementNetDaysUntilExpiration <= 0;
```

Comment:
I think we need to keep using `daysUntilExpiration` in the case where we're looking at the subscription plan details page (when `isSubscriptionPlanDetails` is true).  Otherwise, we won't see the expiration banner on individual plans when the individual plan _has_ expired but there are other plans in the agreement that have _not_ expired.

Reply:
Actually, it looks from the AC of the ticket like we actually don't want what I'm saying, so ignore.